src/Features/CSharp/Portable/Completion/KeywordRecommenders/AssemblyKeywordRecommender.cs

```diff
@@ -39,16 +40,10 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
                     // for that case is necessary check if they Parent is CompilationUnitSyntax
                     case IncompleteMemberSyntax incompleteMember when incompleteMember.Parent is CompilationUnitSyntax:
                         return true;
-                    default:
-                        return false;
                 }
             }
 
-            var skippedTokensTriviaSyntax = token.Parent;
-            // This case happens when:
-            // [$$
-            // namespace Goo {
-            return skippedTokensTriviaSyntax is SkippedTokensTriviaSyntax;
+            return false;
```

Comment:
added bonus.  IDE doesn't have to try to infer what's happening from error recovery.